ucsb_ds_capstone_projects_2021/projects/appfolio/update2.md

```diff
@@ -1,38 +1,52 @@
+![FUNSDpred](https://github.com/DSCapstone2021/ucsb-ds-capstone-2021.github.io/blob/main/ucsb_ds_capstone_projects_2021/projects/appfolio/FUNSDpred.png?raw=true)
+
+(Predictions of LayoutLM for the [FUNSD](https://guillaumejaume.github.io/FUNSD/) dataset, which is similar to the Appfolio Dataset)
 
-Therefore, we have two datasets on hand that we can definitely use (and any other invoice dataset we decide to include for our model fine-tuning, such as the FUNSD dataset) to train our extraction models for total value, and just the Appfolio dataset for extracting the reference IDs of each invoice
+* As an alternative approach, we started our implementation of using object detection to classify key data within the invoice. Object detection is an expansion of image classification, in which rather than simply classifying an image, the detector outputs the localized bounding box coordinates as well as the predicted class. Our team decided to go with [Detectron2](https://detectron2.readthedocs.io/en/latest/tutorials/getting_started.html), Facebook’s open source object detection library. Detectron2’s architecture works by building a feature pyramid network and a region proposal network (RPN) to isolate areas of high interest based on similarities to bounding boxes from the training data. A feature pyramid network works by slowly propagating pixels of bounding boxed images through a neural network, and the region proposal network detects areas of interest from the main photos and feeds it back into the neural network. The RPN makes the Detectron2 algorithm much more efficient, by anchoring images through a sliding window approach. The main reasoning behind the decision to use Detectron2 was based on Detectron2’s capability of achieving high confidence with a relatively simple structure, as well as having ample collections of pretrained models for various object detection algorithms. 
 
-* Worked on combining datasets and cleaning them so as to get them in the format we needed for our models.
+* Worked on cleaning datasets so as to get them in the format we needed for our models (as mentioned before within the individual sections). For detectron2, this included creating bounding boxes around every text of interest and then including the width, height, and label for the bounding box. This is then included with the actual features of the image in a json format. 
 * Implemented a function to visualize the predictions of our models. This is useful in order to see what the label for each bounding box is.
 
 
-![InvoiceLabels](https://github.com/DSCapstone2021/ucsb-ds-capstone-2021.github.io/blob/main/ucsb_ds_capstone_projects_2021/projects/appfolio/InvoiceLabels.png?raw=true)
+![Invoice_Label](https://github.com/DSCapstone2021/ucsb-ds-capstone-2021.github.io/blob/main/ucsb_ds_capstone_projects_2021/projects/appfolio/Invoice_Label.PNG?raw=true)
```

Comment:
Same here

For the image `FUNSDpred.png` in the same directory as the markdown file, you can use `![some-text](FUNSDpred.png)` 
Refer to https://jupyterbook.org/content/figures.html for more info